modules/apps/frontend-theme/frontend-theme-classic/src/css/components/_navbars.scss

```diff
@@ -1,50 +0,0 @@
-.navbar-classic {
-	background-color: #fff;
-
-	&.navbar-light {
-		border-bottom: 1px solid $light;
-	}
-
-	.navbar-form {
-		padding: 0;
-
-		.field-wrapper {
-			right: 0;
-			top: 0;
-
-			.lfr-icon-item {
-				color: #6b6c7e;
-			}
-		}
-	}
-
-	.portlet-search-bar {
-		@include media-breakpoint-down(sm) {
-			width: 100%;
```

Comment:
Removing this I've changed the size of the search on mobile, but honestly why do we need a full-width-search?

![search-2](https://user-images.githubusercontent.com/46778114/107675838-c58b4680-6c98-11eb-8b34-3c5c46795697.jpg)
![search-1](https://user-images.githubusercontent.com/46778114/107675822-c2905600-6c98-11eb-9297-01fa50cd1948.jpg)